src/verifications/verifications.service.ts

```diff
@@ -0,0 +1,131 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { EVM_VERSIONS } from 'src/constants/evm.constants';
+import { PrismaService } from 'src/prisma.service';
+import { VerifyRequestDto } from './verify-request.dto';
+import { Multer } from 'multer';
+import { isAddress } from '@rsksmart/rsk-utils';
+
+@Injectable()
+export class VerificationsService {
+  constructor(private prisma: PrismaService) {}
+
+  getEvmVersions(): string[] {
+    return EVM_VERSIONS;
+  }
+  async verify(data: string, file: Multer.File | undefined) {
+    try {
+      const dataParsed: VerifyRequestDto = JSON.parse(data);
+
+      if (!dataParsed.address) {
+        throw new BadRequestException('Address is required');
+      }
+      if (
+        !isAddress(dataParsed.address) ||
+        !dataParsed.address.includes('0x')
```

Comment:
```suggestion
        !/^0x[a-fA-F0-9]{40}$/.test(dataParsed.address)
```

this also validates the length and that is hex

Reply:
another way would be by using rsk-utils library, https://github.com/rsksmart/rsk-utils/blob/master/src/addresses.ts

Reply:
I am already using isAddress() function from rsk-utils ({ isAddress } from '@rsksmart/rsk-utils';), the only issue is that it does not recognize a wrong address when it does not start by 0x. I realize when writing the unit tests.
Do you want me to use your regex or keep using this util but adding the '0x'? @ivegabr